stdio/bignum.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libphoenix
+ *
+ * bignum.h
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Andrzej Stalke
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+#ifndef _LIBPHOENIX_STDIO_BIGNUM_H
+#define _LIBPHOENIX_STDIO_BIGNUM_H
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#define BIGNUM_DATA_BITS     (8 * sizeof(bignum_data_t))
+#define BIGNUM_DATA_MASK     ((bignum_data_t)(((bignum_ddata_t)1 << BIGNUM_DATA_BITS) - 1))
+#define DOUBLE_EXP_SHIFT     52
+#define DOUBLE_MANTISSA_MASK ((1LLU << DOUBLE_EXP_SHIFT) - 1)
+
+typedef uint32_t bignum_data_t;
+typedef uint64_t bignum_ddata_t;
+
+typedef struct bignum {
+	bignum_data_t *data;
+	size_t len, size;
+} bignum_t;
+
+extern int bignum_init(bignum_t *bn, size_t size, bignum_data_t val);
+extern void bignum_destroy(bignum_t *bn);
```

Comment:
Style: Keep two empty lines separation
```suggestion
extern int bignum_init(bignum_t *bn, size_t size, bignum_data_t val);


extern void bignum_destroy(bignum_t *bn);


```